sympy/testing/tests/test_pytest.py

```diff
@@ -19,7 +19,7 @@ def f():
 def test_lack_of_exception_triggers_AssertionError_callable():
     try:
         raises(Exception, lambda: 1 + 1)
-        assert False
+        raise AssertionError()
     except Failed as e:
         assert "DID NOT RAISE" in str(e)
```

Comment:
This is explicitly testing what happens when you assert something that is false.

Reply:
Oh, actually no it isn't.

Probably it would better to use `else: assert False` to make the test clearer. I don't see why it shouldn't use `assert False` though.